web/src/components/ManagePlan/ManagePlan.js

```diff
@@ -34,28 +35,29 @@ const ManagePlan = ({ plan }) => {
         <Heading>{plan.name}</Heading>
       </Box>
       <Divider my="5" />
-      <Grid templateColumns="5fr 1fr" gridColumnGap={5}>
-        <Stack>
-          <Grid templateColumns="repeat(7, 1fr)" gridColumnGap={2}>
-            {daysOfWeek.map((dayOfWeek, index) => {
-              return (
-                <Box p="3" bg="green.800" key={`dow-${index}`}>
-                  <Text fontSize="md" color="white" textAlign="center">
-                    {dayOfWeek}
-                  </Text>
-                </Box>
-              )
-            })}
-            <PlanWeeks
-              planWeeks={plan.planWeeks}
-              onWorkoutSelected={(workout) => {
-                setSelectedWorkoutId(workout.id)
-              }}
-            />
-          </Grid>
-        </Stack>
-        <WorkoutDetailCell id={selectedWorkoutId} />
-      </Grid>
+      <Flex alignItems="flex-start">
+        <Grid templateColumns="repeat(7, 1fr)" gridColumnGap={2} mx="auto">
```

Comment:
now that the parent container is again a grid, my old problem of not being able to scroll to the entire calendar when it overflows the screen. This is solved by removing `justifyContent="center"` from the parent and adding `mx="auto"` to the children. See "solution #1" in https://stackoverflow.com/a/33455342.